specs/base-exclude-elements.yaml

```diff
@@ -0,0 +1,111 @@
+openapi: "3.0.0"
```

Comment:
Can we use `revision.yaml`?

Reply:
@effoeffi Thank you for your comments!

`revision.yaml` doesn't contain any changes that can be ignored with `--exclude-elements` option, so I created `base-exclude-elements.yaml` from `base.yaml`.

Run oasdiff without `--exclude-elements`
```docker run --rm -t tufin/oasdiff diff --format text https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/base.yaml https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/revision.yaml
### New Endpoints: 1
--------------------
GET /healthcheck  

### Deleted Endpoints: None
---------------------------

### Modified Endpoints: None
----------------------------
```

Run oasdiff with `--exclude-elements`
```
docker run --rm -t tufin/oasdiff diff --format text --exclude-elements description,title,summary https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/base.yaml https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/revision.yaml
### New Endpoints: 1
--------------------
GET /healthcheck  

### Deleted Endpoints: None
---------------------------

### Modified Endpoints: None
----------------------------
```